bmrg-flow/templates/deployment-service.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       {{- if and (eq $k "controller") ($values.workers.security.enable) }}
       serviceAccountName: {{ include "bmrg.name" (dict "context" $context "component" "controller" ) }}
       {{- end }}
-      {{- if $.Values.monitoring.newrelic.apm.key }}
+      {{- if $.Values.monitoring.newrelic.enable }}
```

Comment:
Would this have to be an AND... i.e. if its enabled and there is an APM server key?

Reply:
I guess so yeah, you could have New Relic enabled but you don't use APM.  I can update this.

Reply:
@tlawrie Probably also need to change to and AND on deployment-app.yaml too right?  Do we need to check monitoring.newrelic.browser.key ?

Reply:
Eg. in deployment-app.yaml does the condition need to be:
`{{- if and ($.Values.monitoring.newrelic.enable) ($.Values.monitoring.newrelic.apm.key) ($.Values.monitoring.newrelic.browser.key) }}` ?

Reply:
Hey @tlawrie just checking back in with best way forward here.  Thanks.

Reply:
Yep. Probably. I can't quite remember what is in deployment yaml

Reply:
@tlawrie I've made further updates to Instana and New Relic.  Please take a look when you can.  Thx.